CONTRIBUTING.md

```diff
@@ -1,134 +1,163 @@
 # Develop Temporal Server
+
 This doc is for contributors to Temporal Server (hopefully that's you!)
 
 **Note:** All contributors also need to fill out the [Temporal Contributor License Agreement](docs/development/temporal-cla.md) before we can merge in any of your changes.
 
 ## Prerequisites
 
-### Build prerequisites 
-* [Go Lang](https://golang.org/) (minimum version required is 1.19):
+### Build prerequisites
+
+- [Go Lang](https://golang.org/) (minimum version required is 1.19):
```

Comment:
We are on 1.22 now, but I think it would be better to remove minimum version at all and replace link with "https://go.dev".

Reply:
I will handle this in separate PR.